crates/ruff_python_formatter/src/statement/stmt_global.rs

```diff
@@ -1,12 +1,18 @@
-use crate::{not_yet_implemented, FormatNodeRule, PyFormatter};
-use ruff_formatter::{write, Buffer, FormatResult};
+use ruff_formatter::{format_args, write, Buffer, FormatResult};
 use ruff_python_ast::StmtGlobal;
 
+use crate::prelude::*;
+use crate::{FormatNodeRule, FormattedIterExt, PyFormatter};
+
 #[derive(Default)]
 pub struct FormatStmtGlobal;
 
 impl FormatNodeRule<StmtGlobal> for FormatStmtGlobal {
     fn fmt_fields(&self, item: &StmtGlobal, f: &mut PyFormatter) -> FormatResult<()> {
-        write!(f, [not_yet_implemented(item)])
+        write!(f, [text("global"), space()])?;
+
+        f.join_with(format_args![text(","), space()])
+            .entries(item.names.iter().formatted())
+            .finish()
```

Comment:
Straightforward I think because the grammar is so limited here? These _have_ to be identifiers, and they can't contain any content in-between.